src/createEpicMiddleware.js

```diff
@@ -72,5 +80,18 @@ export function createEpicMiddleware(options = {}) {
     epic$.next(rootEpic);
   };
 
+  epicMiddleware.terminate = () => {
+    if (process.env.NODE_ENV !== 'production' && !store) {
+      require('./utils/console').warn('epicMiddleware.terminate() called before the middleware has been setup by redux. Provide the epicMiddleware instance to createStore() first.');
+    }
+
+    if (!terminating) {
+      terminating = new Promise((resolve) => { resolveTermination = resolve; });
```

Comment:
I think using `Subject` is more elegant here
And type signature in `EpicMiddleware` would be `terminate(): Observable<void>`, it's more consistent for redux-observable

Reply:
- [ ] use-subject